test/CreatorToken.t.sol

```diff
@@ -132,3 +138,63 @@ contract Buying is CreatorTokenTest {
     vm.stopPrank();
   }
 }
+
+contract UpdatingCreatorAndAdminAddresses is CreatorTokenTest {
+  function test_UpdateCreatorAddress(address _newCreator) public {
```

Comment:
Let's extend both of these tests to demonstrate the new creator can subsequently set a new one as well.

Reply:
Extended them in the same commit above: https://github.com/showtime-xyz/creator-tokens/pull/9/commits/e2d09d3b8c2cf5ef735b091b4af37365a82000d1